crates/turbopack-ecmascript-runtime/js/src/shared/runtime-utils.ts

```diff
@@ -99,15 +99,15 @@ function dynamicExport(module: Module, object: Record<string, any>) {
         ) {
           return Reflect.get(target, prop);
         }
-        for (const obj of reexportedObjects) {
```

Comment:
Didn't think you'd need this with `reexportedObjects` being clearly set to a defined value above.